src/Neo.VM/Utility.cs

```diff
@@ -27,6 +27,23 @@ static Utility()
             StrictUTF8.EncoderFallback = EncoderFallback.ExceptionFallback;
         }
 
+        public static bool NotZero(this ReadOnlySpan<byte> x)
```

Comment:
This needs to be moved to the extensions library. For `ReadOnlySpans.cs`

Reply:
> This needs to be moved to the extensions library. For `ReadOnlySpans.cs`

Move it with the rest of the content together?

Reply:
Just that extension method